### ISSUE TITLE
Album support?

### ISSUE THREAD

**Issue by oracle2b** [ubershmekel/redditp#1]
Can you support the viewing of Albums on imgur.com ?

**Comment by ilydlci** [comment]
I just got the following error when trying to display a Imgur album:
>Uncaught TypeError: Cannot read property 'width' of null
>    at script.js:615
>    at Object.convert (EmbedIt.js:47)
>    at Object.embedit.embed (EmbedIt.js:126)
>    at createDiv (script.js:613)
>    at slideBackgroundPhoto (script.js:572)
>    at startAnimation (script.js:484)

Since the Imgur albums convertor calls the `embedFunc` callback with `null`.
I would call this an issue because although albums are not supported, the page should not break (instead it should show some error message and skip the unsupported link).

**Comment by Droyk** [comment]
So what happened dev. will album option ever come?

**Comment by ubershmekel** [comment]
I apologize, I realize albums are important and it's been a while. Especially for subreddits like /r/behindthegifs which are awesome. I would like to modify the way urls are converted to support many more kinds. Something like this: https://github.com/ubershmekel/urlpedia but I'm not yet quite sure about the details.

**Comment by Droyk** [comment]
I can understand that but my question is when you are going to implement it? if you are just going to wait around then it will be too late this feature was requested on 2012 and now it is 2017.....
it's been 6 years and you are still thinking....

**Comment by hppavilion1** [comment]
I think the best way to go about it, if you choose to work on this, might be to (as you said) display the first image, but have a separate redditp-like subapp (under `redditp.com/a/...`, perhaps) that just does a specific imgur album, and it could be opened with a particular hotkey.

**Comment by ubershmekel** [comment]
I agree a hotkey or a button should just allow browsing through the album.

**Comment by hppavilion1** [comment]
I might take a crack at adding imgur album support (just for browsing them at the moment) if you aren't already working on that.

**Comment by Droyk** [comment]
So after 7 years any updates?
